CRM/Contact/Page/View/ContactSmartGroup.php

```diff
@@ -38,7 +38,7 @@ public function browse() {
       }
     }
 
-    $allGroup = CRM_Contact_BAO_GroupContactCache::contactGroup($this->_contactId);
+    $allGroup = CRM_Contact_BAO_GroupContactCache::contactGroup($this->_contactId, FALSE, FALSE);
```

Comment:
Not sure what this is trying to do, but that function only takes 1 parameter:
https://github.com/civicrm/civicrm-core/blob/master/CRM/Contact/BAO/GroupContactCache.php#L425

As regards `CRM_Contact_BAO_GroupContactCache::loadAll()` - see the discussion on https://github.com/civicrm/civicrm-core/pull/26357 about deprecating it.